### PR TITLE
bugfixes

### PR DESCRIPTION
**Pull request by xithan**
- broken toolbar icons
- footer in wrong place

### CODE REVIEW

File: card/mod/03_machines/lib/javascript/wagn_mod.js.coffee [hunk 1]
```diff
@@ -176,8 +176,11 @@ $.extend wagn,
 # sidebars
 
 wrapDeckoLayout = () ->
-  $('body > article, body > aside, body > footer').wrapAll('<div class="container"/>')
+  $footer  = $('body > footer').first()
+  $('body > article, body > aside').wrapAll('<div class="container"/>')
   $('div.container > article, div.container > aside').wrapAll('<div class="row row-offcanvas">')
+  if $footer
+    $('body').append $footer
```
Comment (ethn) [inline card/mod/03_machines/lib/javascript/wagn_mod.js.coffee]: were there some pages without a footer breaking?

File: card/mod/05_standard/set/all/rich_html/toolbar.rb [hunk 2]
```diff
@@ -233,7 +233,8 @@ def toolbar_button text, symbol, hide=nil, tag_args={}
     tag_args[:class] = [tag_args[:class], 'btn btn-primary'].compact * ' '
     tag_args[:title] ||= text
     link_text = glyphicon symbol
-    link_text += "<span class='menu-item-label #{hide}'>#{text}</span>"
+    link_text += content_tag :span, text.html_safe,
+                             class: "menu-item-label #{hide}"
```
Comment (ethn) [inline card/mod/05_standard/set/all/rich_html/toolbar.rb]: I added a similar fix last night, but this is cleaner.

**Comment by ethn** [comment]
maybe you've already figured this out, but it looks to me like semaphore doesn't run when a PR won't automatically merge.  (I wish it wouldn't show the check mark in that case)

File: card/mod/03_machines/lib/javascript/wagn_mod.js.coffee (hunk 1 above)
Comment (xithan) [inline card/mod/03_machines/lib/javascript/wagn_mod.js.coffee]: without this the content was moved below the footer

**Pull request merged**